db/migrate/20170227224833_add_class_room_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddClassRoomToUsers < ActiveRecord::Migration
+  def change
+    add_reference :users, :class_room, index: true, foreign_key: true
+  end
```

Comment:
Class room is now a reference to another table -- but in the form it's a text field

```
    <%= f.label :class_room %><br>
<%= f.text_field :class_room %><br>
```
https://github.com/vinithanatarajan/teacher-portal/blob/master/app/views/students/_form.html.erb#L7

How is the class room saved? 

The table looks like this:
```
  create_table "class_rooms", force: :cascade do |t|
    t.datetime "created_at", null: false
    t.datetime "updated_at", null: false
end
```

There's no actual fields on the table.